docs/pages/versions/unversioned/expokit/expokit.md

```diff
@@ -80,28 +80,33 @@ ExpoKit's release cycle follows the Expo SDK release cycle. When a new version o
 
 ### iOS
 
-- Open up `ios/Podfile` in your project, and update the `ExpoKit` tag to point at the [release](https://github.com/expo/expo/releases) corresponding to your SDK version. Run `pod update` then `pod install`.
+- Go to https://expo.io/--/api/v2/versions and find the `expokitNpmPackage` key under `sdkVersions.[NEW SDK VERSION]`.
+- Update your version of expokit in `package.json` to the version in `expokitNpmPackage` and run `yarn` or `npm install`.
```

Comment:
`expokit` package is covered in Android section of the instructions, it doesn't affect the iOS project

Reply:
> it doesn't affect the iOS project
Oh really? Is it because we take it directly from https://expo.io/--/api/v2/version?
So in that way should we leave `"expokit": "32.0.6"` in our package.json for iOS, or just delete it?

Reply:
If you'd have a project that is ejected and you would care only about iOS you could remove `expokit` from your `package.json`.